internal/discover/graphics.go

```diff
@@ -56,6 +56,9 @@ func NewGraphicsMountsDiscoverer(logger logger.Interface, driver *root.Driver, n
 		driver.Root,
 		[]string{
 			"libnvidia-egl-gbm.so.*",
+			"gbm/nvidia-drm_gbm.so",
+			"libnvidia-egl-wayland.so.*",
+			"libnvidia-vulkan-producer.so*",
```

Comment:
We want:
```suggestion
			"libnvidia-vulkan-producer.so.*",
```
and need to create the `.so` symlink if required as a hook.